packages/component-library/components/Paragraph/Paragraph.tsx

```diff
@@ -0,0 +1,46 @@
+import classNames from "classnames"
+import { createElement } from "react"
+
+const styles = require("./Paragraph.module.scss")
```

Comment:
Where _is_ `Paragraph.module.scss`. It's not in master and I'm not seeing it in this PR (but then how is the build passing 🤔 )

Reply:
The Elm component included the styles. Since that PR is now merged, the module should be in master and hence the file diff is removed in this PR. 

Elm PR: https://github.com/cultureamp/kaizen-design-system/pull/320/files
Paragraph.module.css in master: https://github.com/cultureamp/kaizen-design-system/blob/master/packages/component-library/components/Paragraph/Paragraph.module.scss